app/components/bali/avatar/component.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Bali
+  module Avatar
+    class Component < ApplicationViewComponent
+      renders_one :picture, Picture::Component
+
+      def initialize(form:,
+                     attr_model_name:,
+                     accepted_formats:,
+                     placeholder_url: 'https://bulma.io/images/placeholders/256x256.png',
```

Comment:
I don't think it's a good idea to rely on an image on a server that we don't control. If for some reason bulma decides to share the path or the server is down or whatever other reason, we won't find out until it's broken.

We should be embedding the placeholder image within the project.